static/style.css

```diff
@@ -6,4 +6,29 @@ h1{
     font-family:Arial;
     color:rgb(0, 0, 0);
     font-size: 28;
-}
```

Comment:
Extra curly brace

Reply:
I don't understand this part, what do you mean by extra curly brace?
Do I have to add a extra curly brace?